example/disk-agent.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+CLUSTER=${CLUSTER:-local}
+IC_PORT=${IC_PORT:-29012}
+GRPC_PORT=${GRPC_PORT:-9001}
+MON_PORT=${MON_PORT:-8769}
+
+find_bin_dir() {
+    readlink -e `dirname $0`
+}
+
+BIN_DIR=`find_bin_dir`
+source ./prepare_binaries.sh || exit 1
+
+diskagentd \
+    --domain             Root \
+    --node-broker        localhost:$GRPC_PORT \
+    --ic-port            $(( $IC_PORT + $1 * 100 )) \
```

Comment:
Copy-paste gone wrong... Either you shouldn't assume 1st argument is an agent's number, or you should cut it from argument array $@

I'd suggest to make a separate function and use it for both this script and original start_nbs_storage

Reply:
I decided to just move everything to env variables with default values